Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -994,20 +967,19 @@
           }
         },
         "Start": 10,
-        "Length": 19
+        "Length": 16
       }
     ]
   },
   {
-    "Input": "De les is van 11:01 tot 11",
+    "Input": "De les is van 11:01 's ochtends tot 11",
```

Comment:
What happens if "morning" is not specified? That case should also be supported and have a spec for it.

Reply:
added spec.
"van 11:01 's ochtends tot 11" is parsed as (T11:01,T23,PT11H59M)
"van 11:01 tot 11" is parsed as (T11:01,T11,PT23H59M)
The same happens in English if we remove "am" from the case "from 11:01 am to 11"